gcloud/Gemfile

```diff
@@ -35,14 +35,6 @@ gem "google-cloud-video_intelligence",
     path: "../google-cloud-video_intelligence"
 gem "google-cloud-vision", path: "../google-cloud-vision"
 
-if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.1")
-  # WORKAROUND: builds are failing on Ruby 2.0.
-  # We think this is because of a bug in Bundler 1.6.
-  # Specify a viable version to allow the build to succeed.
-  gem "jwt", "~> 1.5"
-  gem "kramdown", "< 1.17.0" # Error in yard with 1.17.0
-end
-
 # WORKAROUND: builds are having problems since the release of 3.0.0
 # pin to the last known good version
 gem "public_suffix", "~> 2.0"
```

Comment:
I think we can also remove the pinned `public_suffix` gem as well, since that was mostly around older ruby support as well.

Reply:
I'll do that. Are we pinning rake to 11.x for the same reason? Can we update to 12?